docs/pages/management/export-audit-events/splunk.mdx

```diff
@@ -0,0 +1,783 @@
+---
+title: Monitor Teleport Audit Events with Splunk
+description: How to configure Teleport's Event Handler plugin to send audit logs to Splunk
+---
+
+Teleport's Event Handler plugin receives audit logs from the Teleport Auth
+Service and forwards them to your log management solution, letting you perform
+historical analysis, detect unusual behavior, and form a better understanding of
+how users interact with your Teleport cluster.
+
+In this guide, we will show you how to configure Teleport's Event Handler plugin
+to send your Teleport audit logs to Splunk. In this setup, the Event Handler
+plugin forwards audit logs from Teleport to Splunk's Universal Forwarder,
+which stores them in Splunk Cloud Platform or Splunk Enterprise for
+visualization and alerting. 
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
```

Comment:
This works for OSS and ent right?  I don't think there's anything specific that this only works for ent.

Reply:
Do plugins also work for OSS? My understanding was that plugins are a commercial feature. If not, we'll need to update all of our plugin guides.

Reply:
Yes @ptgott , they work for OSS too.  They have some different behavior in that for Access Requests plugins they'll give CLI output instead of a link to the web access requests that are ent only.